webhook/README.md

```diff
@@ -0,0 +1,21 @@
+# Prismic Webhook
+
+This is a Lambda which expects to live at a [Function URL](https://docs.aws.amazon.com/lambda/latest/dg/lambda-urls.html) which is configured as a webhook in Prismic.
+
+It receives Webhook events, checks that they are valid and contain a specified secret, and then forwards them to an EventBridge bus.
+
+We configure one of these Lambdas with a webhook that runs on document unpublish, and the events are subscribed to by an "unpublisher" Lambda in each pipeline stack.
```

Comment:
We should definitely use "unpublish" because it's the Prismic nomenclature, but let's add a brief hint for people who aren't familiar with weird Prismic naming:

```suggestion
We configure one of these Lambdas with a webhook that runs on document unpublish (i.e. delete), and the events are subscribed to by an "unpublisher" Lambda in each pipeline stack.
```